recon-enumeration/slides/slides.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "SSS: Session: Recon & Enumeration"
+revealOptions:
+  background-color: 'aquamarine'
+  transition: 'none'
+---
+
+# Recon & Enumeration & Recap
+
+SSSv9
+
+---
+
+## Phases of Penetration Testing
+
+penetration testing / pentester
+
+1. reconnaissance - gathering information about the target system: technologies, subdomains, open ports, Google hacking
+2. scanning - manually or automatically to discover vulnerabilities (XSS, SQLi)
+3. gaining access - exploiting the found vulnerabilities using enumeration
+4. maintaining access - backdoors
+5. covering tracks - remove any evidence of the attack
+
+----
+
+## 1. Reconnaissance
```

Comment:
From what I some here you have `1. Reconnaissance`, but for the other headers you don't use the numbers, only for the third one. I think it's better to use only the title.
```suggestion
## Reconnaissance
```